### PR TITLE
Added index html file

### PR DESCRIPTION
**Pull request by rohita-ch**

### CODE REVIEW

**Review by smandalsid — APPROVED** [review]
very good coding. very bad coding.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
```
Comment (PGTR11) [inline index.html:8]: edit head

**Review by PGTR11 — APPROVED** [review]
great you can upload

File: index.html (hunk 1 above)
Comment (rohita-ch) [reply index.html:8]: sure

**Pull request merged**